pages/cuidado-compartilhado/raps-hospitalar/index.js

```diff
@@ -58,67 +61,147 @@ const RapsHospitalar = ({ }) => {
         titulo="<strong>Internações finalizadas desde o início do ano</strong>"
       />
 
-      { InternacoesRapsAltas12m
+      { internacoesRapsAltas12m
         ? <GraficoInfo
-          descricao={ `Internações finalizadas entre ${InternacoesRapsAltas12m["a_partir_de_mes"]} de ${InternacoesRapsAltas12m["a_partir_de_ano"]} e ${InternacoesRapsAltas12m["ate_mes"]} de ${InternacoesRapsAltas12m["ate_ano"]}` }
+          tooltip="São consideradas as internações de munícipes em UPAs e hospitais gerais ou psiquiátricos, vinculados a qualquer esfera federativa, desde que o tipo de leito, o procedimento principal ou o CID diagnóstico sejam referentes a questões de saúde mental. &quot;Antes&quot; inclui internados com acompanhamento prévio na RAPS, com registros de ações psicossociais (RAAS) ou boletins de produção ambulatorial (BPA) nos 6 meses anteriores à internação. &quot;Depois&quot; abrange egressos da internação na rede hospitalar com altas relacionadas à saúde mental e registros na RAPS de ações psicossociais (RAAS) ou boletins de produção ambulatorial (BPA) no mesmo mês da alta ou no mês seguinte."
```

Comment:
Corrige esse typo, por favor?

```suggestion
          tooltip="São consideradas as internações de munícipios em UPAs e hospitais gerais ou psiquiátricos, vinculados a qualquer esfera federativa, desde que o tipo de leito, o procedimento principal ou o CID diagnóstico sejam referentes a questões de saúde mental. &quot;Antes&quot; inclui internados com acompanhamento prévio na RAPS, com registros de ações psicossociais (RAAS) ou boletins de produção ambulatorial (BPA) nos 6 meses anteriores à internação. &quot;Depois&quot; abrange egressos da internação na rede hospitalar com altas relacionadas à saúde mental e registros na RAPS de ações psicossociais (RAAS) ou boletins de produção ambulatorial (BPA) no mesmo mês da alta ou no mês seguinte."
```